lms/static/scripts/frontend_apps/components/dashboard/HomeActivity.tsx

```diff
@@ -21,9 +21,9 @@ type CoursesTableRow = {
 const courseURL = (id: number) => urlPath`/courses/${String(id)}`;
 
 /**
- * List of courses that belong to a specific organization
+ * List of courses current user is teaching or is an admin on
```

Comment:
```suggestion
 * List of courses that current user has access to in the dashboard
```

A user can have access because:

- They are an instructor
- They are an institutional administrator
- They are Hypothesis staff that have been granted access

Since the exact reason doesn't affect the frontend, I'd suggest we avoid this comment getting out of date by just saying that they are courses the user has access to.